guides/common/modules/proc_registering-a-host-using-the-bootstrap-script.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Registering_a_Host_Using_the_Bootstrap_Script_{context}"]
-= Registering a Host to {ProjectName} Using The Bootstrap Script
+= Registering Hosts by Using The Bootstrap Script
```

Comment:
Unsure if this change is necessary. Does it "align titles nicely" or what is the reasoning behind it?

Reply:
Just to align the titles.